bin/update_clinvar_schema.py

```diff
@@ -7,7 +7,7 @@
 import os
 
 SCHEMA_LOCATION_ATTRIBUTE = 'SchemaLocation'
-JAVA_SOURCE_CODE_DIRECTORY = 'clinvar-parser/src/main/java'
+JAVA_SOURCE_DIR = 'clinvar-xml-parser/src/main/java'
```

Comment:
This may only work if run from the project root folder. In order to point to the Java folder from this one, you can get the Python script folder [this way](https://stackoverflow.com/a/19323023)